drivers/adc/adc_stm32.c

```diff
@@ -1046,18 +1047,18 @@ static int adc_stm32_init(const struct device *dev)
 		channel_offset = 0UL;
 	} else if (adc == ADC2) {
 		channel_offset = 8UL;
-	} else   /*Case ADC3*/ {
+	} else { /*Case ADC3*/
 		channel_offset = 16UL;
 	}
 	/* Read factory calibration factors */
 	for (uint32_t count = 0UL; count < ADC_LINEAR_CALIB_REG_COUNT; count++) {
 		linear_calib_buffer = *(uint32_t *)(
 			ADC_LINEAR_CALIB_REG_1_ADDR + channel_offset + count
-		);
+			);
```

Comment:
While you're at, I think this break is superfluous and could be removed.
(in a dedicated commit, if you don't want to break the uncrustify only commit)